qbertclient/request_utils.py

```diff
@@ -48,18 +48,18 @@ def session_with_retries(host, max_retries=10):
     return session
 
 
-def make_req(session, endpoint, method, body, verify=True):
+def make_req(session, endpoint, method, body, **kwargs):
     """
     Main request wrapper
     :param session:
     :param endpoint:
     :param method:
     :param body:
-    :param verify:
+    :param kwargs:
     :return:
     """
-    resp = session.request(method, endpoint, json=body, verify=verify,
```

Comment:
I don't see anyone making verify=False request to make_req, am I missing something

Reply:
Are you expecting client to specify verify=False ?

Reply:
Yes, the client should pass in if they want to control this attribute. They need to do it anyway. I changed the code to transparently pass these arguments down to the requests library. They should be able to pass any other arguments as well with this change.
The requests library defaults the verify field to be True if the user doesn't pass this attribute.